.fixtures.yml

```diff
@@ -3,3 +3,5 @@ fixtures:
     facts:
```

Comment:
OOps, i messed up and copy pasted in facts (that should be ruby_task_helper). We should get the module from the forge once we have a release.